source/elements/oneDAL/source/introduction.rst

```diff
@@ -2,19 +2,19 @@
 Introduction
 ============
 
-oneDAL is library that provides building blocks covering all stages of data analytics: data acquisition
- from a data source, preprocessing, transformation, data mining, modeling, validation, and decision making.
+oneDAL is a library that provides building blocks covering all stages of data analytics: data acquisition
+from a data source, preprocessing, transformation, data mining, modeling, validation and decision making.
 
-Library consists of the following major components: Data Management, Algorithms, and Services
+Library consists of the following major components: Data Management, Algorithms and Services
```

Comment:
Library consists of the following major components: Data Management, Algorithms and Services

The Data Management component includes classes and utilities for data acquisition, initial preprocessing and normalization, for data conversion into numeric formats done by one of supported Data Sources and for model representation.

We still need a better structuring or connection between 'Data Management' in the first sentence and 'Data Management' in the second and other paragraph

Reply:
we will have just component mention with link to corresponding section. Second mention have been moved to corresponding sections as headers